demo/kubernetes/rook-storageclass.yaml

```diff
@@ -17,12 +17,9 @@ kind: StorageClass
 metadata:
    name: rook-block
    namespace: rook
-provisioner: kubernetes.io/rbd
+provisioner: rook.io/rookblock
```

Comment:
Since this is not a tpr, i don't think we need the redundant name `rookblock`. How about if we use `rook.io/block`?